pkg/descriptions.json

```diff
@@ -71,5 +71,10 @@
             "System.ServiceModel.UpnEndpointIdentity",
             "System.ServiceModel.SpnEndpointIdentity"
         ]
+    },
+    {
+        "Name": "svcutilcore",
+        "Description": "Generates Serialization code for all data types used in service contracts in the compiled assembly that can be serialized using the XmlSerializer",
```

Comment:
Does "data types" mean something specific?

I don't think we need to say "compiled assembly". All assemblies are compiled. We may allow the tool to work against any input assembly that does not have to be the current "compiled" assembly.

It is technically inaccurate to say "assembly that can be serialized using the XmlSerializer". It's "types" that can be serialized using the XmlSerializer.